src/utils.js

```diff
@@ -15,12 +15,16 @@ export function classList (props) {
     'fa-border': props.border,
     'fa-li': props.listItem,
     'fa-inverse': props.inverse,
+    'fa-flip': props.flip,
```

Comment:
This probably needs to be
```suggestion
    'fa-flip': props.flip === true,
```

Otherwise since a string is truthy I think anytime the value is `horizontal` or `vertical` or `both` you'll get the `fa-flip` class applied.

Reply:
You were/are right about the failing test @robmadole without having this... I actually saw my icons continuously rotating and ignored it for whatever reason.  This addition, however, does prevent the icons from flipping yet allowing them to have been "rotated" as desired.